charts/grafana/Chart.yaml

```
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: grafana
-version: 6.0.1
-appVersion: 7.2.1
+version: 6.0.2
```

Shouldn't we bump the minor as well?
```suggestion
version: 6.1.0
```

---

agree and done